prov/verbs/src/verbs_info.c

```diff
@@ -616,7 +617,8 @@ static inline int fi_ibv_get_qp_cap(struct ibv_context *ctx,
 	info->tx_attr->size	 	= init_attr.cap.max_send_wr;
 
 	info->rx_attr->iov_limit 	= init_attr.cap.max_recv_sge;
-	info->rx_attr->size	 	= init_attr.cap.max_recv_wr;
+	info->rx_attr->size	 	= MIN(init_attr.cap.max_recv_wr,
+						device_attr->max_qp_wr);
```

Comment:
I agree that this looks strange but init_attr.cap.max_recv_wr is icremented after ibv_create_qp call and the next qp creation is failing with corresponding message about the new value.
I see this only on my quite old iWapr HW